spec/hardware/virtualization.fmf

```diff
@@ -38,4 +38,4 @@ example:
 link:
   - implemented-by: /tmt/steps/provision/artemis.py
   - implemented-by: /tmt/steps/provision/mrack.py
-    note: "``virtualization.is-virtualized`` only"
+    note: "``virtualization.is-virtualized`` ``virtualization.hypervisor`` only"
```

Comment:
Please, add also `versionchanged` recording when `virtualization.hypervisor` was added to mrack plugin, similar to as https://github.com/teemtee/tmt/pull/2733/files#diff-844c430661f020411c529315cabf0d4099728fc3226a2b448ceacfc6637fe550R4

(We should have a checklist for extending HW requirement support in a plugin, we always forget something a PR needs to have...)